### PR TITLE
fix: MD005/list-indent

### PR DESCRIPTION
**Pull request by nschonni**
- Replace mixed tab indenting
- Trim trailing spaces
- Remove blockquote spacing
- Remove interior list indenting
- Spacing around headers and code fences

### CODE REVIEW

**Comment by mairaw** [comment]
I did a commit to make the content validation pass.

File: docs/framework/windows-workflow-foundation/whats-new-in-wf-in-dotnet.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-﻿---
+---
```
Comment (nschonni) [inline docs/framework/windows-workflow-foundation/whats-new-in-wf-in-dotnet.md]: Was it a BOM or something?

Comment (nschonni) [reply docs/framework/windows-workflow-foundation/whats-new-in-wf-in-dotnet.md]: Ah, didn't see the commit message

Comment (mairaw) [reply docs/framework/windows-workflow-foundation/whats-new-in-wf-in-dotnet.md]: Yep 😄 I've asked them to change the error message in these cases to be more explicit. Not sure when it's going to be implemented.

File: docs/framework/configure-apps/file-schema/windows-identity-foundation/identityconfiguration.md [hunk 2]
```diff
@@ -4,89 +4,95 @@ ms.date: "03/30/2017"
...
-1.  If no `<identityConfiguration>` element is specified. A default identity configuration is created at runtime and populated with default values.  
-  
-2.  If a single `<identityConfiguration>` element is specified. It is the default identity configuration. It does not matter whether it is named or unnamed.  
-  
-3.  If multiple `<identityConfiguration>` elements are specified. The unnamed element specifies the default identity configuration. It is recommended that when you specify multiple `<identityConfiguration>` elements, one of them should be unnamed.  
-  
+
+Specifies service-level identity settings.
+
+ \<system.identityModel>
```
Comment (mairaw) [inline docs/framework/configure-apps/file-schema/windows-identity-foundation/identityconfiguration.md]: This change removes the new line between the two. 
image

Maybe do this?

```suggestion
 \<system.identityModel>\
```

File: docs/framework/windows-workflow-foundation/whats-new-in-wf-in-dotnet.md (hunk 1 above)
Comment (nschonni) [reply docs/framework/windows-workflow-foundation/whats-new-in-wf-in-dotnet.md]: Soonish maybe https://github.com/MicrosoftDocs/azure-docs/issues/14166#issuecomment-459082416

This is how I found to clean it up before https://github.com/MicrosoftDocs/azure-docs/pull/10954#issue-198220272

**Review by mairaw — CHANGES_REQUESTED** [review]
Changes look good @nschonni. Left just one comment.

**Review by mairaw — APPROVED** [review]
Looks good now. Thanks again!

**Pull request merged**